### ISSUE TITLE
NTR: 4-hydroxyphenylactic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8626]
New term request 4-hydroxyphenylactic acid high in urine (segal_050105154527): Aciduria  UMLS: 1841664

**Comment by pnrobinson** [comment]
Is this intended to be a xenobiotic? What is the disease context?
https://en.wikipedia.org/wiki/4-Hydroxyphenylacetic_acid

**Comment by pnrobinson** [comment]
@MickeySegal
updates?

**Comment by MickeySegal** [comment]
https://omim.org/entry/140350
Her urine contained large amounts of 4-hydroxyphenylpyruvic acid, 4-hydroxyphenylactic acid, and 4-hydroxyphenylacetic acid.

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001004
- label: Elevated urine hydroxyphenyllactic acid level
- parents: [Abnormal urinary non-proteinogenic amino acid level (HP:0034027)](https://hpo.jax.org/app/browse/term/HP:0034027)
- pmid: PMID:4720815
- def: The amount of hydroxyphenyllactic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: Hydroxyphenyllactic acid or 4-hydroxyphenyllactate (the L-form) is a tyrosine metabolite. The level of L-hydroxyphenyllactic acid is elevated in patients with a deficiency of the enzyme p-hydroxyphenylpyruvate oxidase (EC 1.14.2.2).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8626
- synonyms:

**Comment by MickeySegal** [comment]
I agree on the fixed spelling.
